<launch>
    <!-- パラメータ設定セクション nodeとは独立させて書く-->
    <!-- param name="directory_name" value="/home/rokuto/result_1119449"/ -->
    <param name="directory_name" value="/home/rokuto/result_29141627"/>
    <param name="max_distance" value="80"/>
    <param name="z_min" value="-1.5"/>
    <param name="z_max" value="1.0"/>
    <param name="x_divide" value="20"/>
    <param name="y_divide" value="20"/>
    <node name="visualizer" pkg="spoofing_sim" type="heatmap_visualize.py" output="screen" />
</launch>

